code/datums/actions/action.dm

```diff
@@ -327,7 +327,7 @@
  */
 /datum/action/proc/update_button_status(atom/movable/screen/movable/action_button/current_button, force = FALSE)
 	current_button.update_keybind_maptext(full_key)
-	if(IsAvailable())
+	if(IsAvailable(feedback = FALSE))
```

Comment:
The other review was correct, this code is doing nothing

The reason your pr fixes the aforementioned bug is...

Reply:
Alright, I removed it then.